archive/d/dart/prime-number.dart

```diff
@@ -0,0 +1,38 @@
+// Issue 4970
+import 'dart.math';
```

Comment:
I don't think this is the correct way to specify this import. Take a look at some of the [Dart samples](https://github.com/TheRenegadeCoder/sample-programs/tree/main/archive/d/dart).